package.json

```diff
@@ -43,10 +43,10 @@
   "scripts": {
     "build": "node --harmony script/build",
     "build-watch": "node --harmony script/build -- --watch",
-    "test": "cross-env NODE_ENV=development nyc mocha --harmony --recursive --require script/test && cross-env NODE_ENV=production nyc mocha --harmony --recursive --require script/test",
+    "test": "cross-env NODE_ENV=development nyc mocha --harmony --recursive --require script/test && cross-env NODE_ENV=production nyc --no-clean mocha --harmony --recursive --require script/test",
     "test-watch-dev": "cross-env NODE_ENV=development mocha --harmony --recursive --require script/test --watch",
     "test-watch-prod": "cross-env NODE_ENV=production mocha --harmony --recursive --require script/test --watch",
-    "test-debug-dev": "cross-env NODE_ENV=development mocha --harmony --recursive --require script/test",
+    "test-debug-dev": "mocha --harmony --recursive --require script/test",
```

Comment:
Will this still run the development build?

Reply:
No that was a bug .. thanks for the info ... is fixed now